modules/cnf-configuring-nrop-on-schedlable-control-planes.adoc

```diff
@@ -0,0 +1,219 @@
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output:
+
+[source,terminal]
+----
+NAME                STATUS   ROLES                         AGE     VERSION
+worker-0            Ready    worker,worker-cnf             100m    v1.33.3
+worker-1            Ready    worker                        93m     v1.33.3
+master-0            Ready    control-plane,master,worker   108m    v1.33.3
+master-1            Ready    control-plane,master,worker   107m    v1.33.3
+master-2            Ready    control-plane,master,worker   107m    v1.33.3
+worker-2            Ready    worker                        100m    v1.33.3
+----
+
+. Verify that the NUMA Resources Operator’s pods are running on the intended nodes by running the following command. You should see a numaresourcesoperator pod for each node group you specified in the CR:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-numaresources -o wide
+----
++
+.Example output:
+[source,terminal]
+----
+NAME                                               READY   STATUS    RESTARTS   AGE     IP            NODE       NOMINATED NODE   READINESS GATES
+numaresources-controller-manager-bdbdd574-xx6bw    1/1     Running   0          49m     10.130.0.17   master-0   <none>           <none>
+numaresourcesoperator-master-lprrh                 2/2     Running   0          20m     10.130.0.20   master-0   <none>           2/2
+numaresourcesoperator-master-qk6k4                 2/2     Running   0          20m     10.129.0.50   master-2   <none>           2/2
+numaresourcesoperator-master-zm79n                 2/2     Running   0          20m     10.128.0.44   master-1   <none>           2/2
+numaresourcesoperator-worker-cnf-gqlmd             2/2     Running   0          4m27s   10.128.2.21   worker-0   <none>           2/2
+----
+
+. Confirm that the NUMA Resources Operator has collected and reported the NUMA topology data for all nodes in the specified groups by running the following command:
++
+[source,terminal]
+----
+$ oc get noderesourcetopologies.topology.node.k8s.io
+----
++
+.Example output:
+[source,terminal]
+----
+NAME          AGE
+worker-0      6m11s
+master-0      22m
+master-1      21m
+master-2      21m
+----
++
+The presence of a NodeResourceTopology resource for a node confirms that the NUMA Resources Operator was able to schedule a pod on it to collect the data, enabling topology-aware scheduling.
+
+
+. Inspect a single Node Resource Topology by running the following command:
++
+[source,terminal]
+----
+$ oc get noderesourcetopologies <master_node_name> -o yaml
+----
++
+.Example output:
+[source,yaml]
+----
+apiVersion: topology.node.k8s.io/v1alpha2
+attributes:
+- name: nodeTopologyPodsFingerprint
+  value: pfp0v001ef46db3751d8e999
+- name: nodeTopologyPodsFingerprintMethod
+  value: with-exclusive-resources
+- name: topologyManagerScope
+  value: container
+- name: topologyManagerPolicy
+  value: best-effort
```

Comment:
it's not big deal but let's just replace "best-effort" with "single-numa-node" to make the example match the TAS criteria